js/samples/ember-sample/package.json

```diff
@@ -76,6 +76,7 @@
     "minimatch": "3.0.5",
     "minimist": "^1.2.7",
     "socket.io": "^4.5.4",
+    "socket.io-parser": "^4.2.3",
```

Comment:
why do we even need this package?

Reply:
it was installed while updating the version

Reply:
removed, not needed